esp-wifi/Cargo.toml

```diff
@@ -32,16 +32,27 @@ toml-cfg.workspace = true
 libm.workspace = true
 cfg-if.workspace = true
 
+[dev-dependencies]
+esp-println = { workspace = true, features = ["log"] }
+esp-backtrace = { workspace = true }
+embedded-svc.workspace = true
+embassy-executor = { workspace = true }
+embassy-time.workspace = true
+embassy-futures.workspace = true
+futures-util.workspace = true
+embassy-net = { workspace = true }
+embassy-sync.workspace = true
+
 [features]
 default = [ "utils", "log" ]
 
 # chip features
-esp32c2 = [ "esp32c2-hal", "esp-wifi-sys/esp32c2" ]
-esp32c3 = [ "esp32c3-hal", "esp-wifi-sys/esp32c3" ]
-esp32c6 = [ "esp32c6-hal", "esp-wifi-sys/esp32c6" ]
-esp32 = [ "esp32-hal",  "esp-wifi-sys/esp32" ]
-esp32s2 = [ "esp32s2-hal", "esp-wifi-sys/esp32s2" ]
-esp32s3 = [ "esp32s3-hal", "esp-wifi-sys/esp32s3" ]
+esp32c2 = [ "esp32c2-hal", "esp-wifi-sys/esp32c2", "esp-println/esp32c2", "esp-backtrace/esp32c2", "embassy-executor/arch-riscv32" ]
+esp32c3 = [ "esp32c3-hal", "esp-wifi-sys/esp32c3", "esp-println/esp32c3", "esp-backtrace/esp32c3", "embassy-executor/arch-riscv32" ]
+esp32c6 = [ "esp32c6-hal", "esp-wifi-sys/esp32c6", "esp-println/esp32c6", "esp-backtrace/esp32c6", "embassy-executor/arch-riscv32" ]
+esp32 = [ "esp32-hal",  "esp-wifi-sys/esp32", "esp-println/esp32", "esp-backtrace/esp32", "embassy-executor/arch-xtensa" ]
```

Comment:
> "embassy-executor/arch-xtensa"

I forgot, is this still necessary with the executors in esp-hal?

Reply:
Good spot, added to the list :)